configstore/index.d.ts

```diff
@@ -1,37 +1,65 @@
-// Type definitions for configstore 0.3.0
+// Type definitions for Configstore 2.1.0
 // Project: https://github.com/yeoman/configstore
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
```

Comment:
Please add your name to the end: `// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, ArcticLight <https://github.com/ArcticLight/>`, and keep the 4th line of the header.